frontend/src/components/common/loader/loader.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  size?: number;
+  centered?: boolean;
+  className?: string;
+};
+
+const Loader: FC<Props> = ({ size = 80, centered = true, className }) => {
+  return (
+    <div
+      className={`${styles['lds-ring']} ${centered ? styles['centered'] : ''} ${className}`}
```

Comment:
Refactor it.... If needed complex logic to compose different classNames probably should use [classnames](https://www.npmjs.com/package/classnames)